src/actions/index.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable arrow-parens */
+
+export const createBook = book => ({
+  type: 'CREATE_BOOK',
+  payload: book,
+});
+
+export const removeBook = book => ({
+  type: 'REMOVE_BOOK',
+  payload: book,
```

Comment:
[Optional]

We could return only `bookId` of `book` for the `payload` property instead.